tilelang/quantize/mxfp.py

```
@@ -86,7 +195,36 @@ def get_mxfp_intrin_group(
     assert storage_dtype in [T.int32, T.int8, T.uint8], f"Invalid storage_dtype: {storage_dtype}. Expected 'int32' or 'int8' or 'uint8'."
 
+    # Detect AMD gfx950 target to select the HIP C++ dequantization implementation.
+    # All other targets (NV, RDNA, MI300) use the default CUDA PTX path below.
+    _is_gfx950 = False
+    if target is not None:
+        try:
+            from tilelang.utils.target import target_is_gfx950
+
+            _is_gfx950 = target_is_gfx950(target)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)